spirv_reflect.c

```diff
@@ -1840,62 +1859,64 @@ static SpvReflectResult ParseDescriptorType(SpvReflectShaderModule* p_module)
     SpvReflectDescriptorBinding* p_descriptor = &(p_module->descriptor_bindings[descriptor_index]);
     SpvReflectTypeDescription* p_type = p_descriptor->type_description;
 
-    switch (p_type->type_flags & SPV_REFLECT_TYPE_FLAG_EXTERNAL_MASK) {
-      default: assert(false && "unknown type flag"); break;
+    if (p_descriptor->descriptor_type == INVALID_VALUE) {
```

Comment:
I didn't modify anything here; just indented the whole `switch` statement. Enabling "Hide whitespace changes" in the settings will make it easier to review.